tests/functional/cli/test_cli_repository.py

```diff
@@ -45,6 +49,49 @@ def test_list_all_commits(gitlab_cli, project):
     assert len(ret_all.stdout) > len(ret.stdout)
 
 
+def test_commit_merge_requests(gitlab_cli, project, merge_request, wait_for_sidekiq):
+    """This tests the `project-commit merge-requests` command and also tests
+    that we can print the result using the `json` formatter"""
```

Comment:
Just an idea, if we're testing mostly the list output, we could simplify this test a lot by not relying on MRs and just getting the diff of the HEAD: `gitlab -o json project-commit diff --project-id project.id --id project.default_branch`. This command has the same issue.

Reply:
Sure. But it does allows us to test both `project-commit merge-requests` and the list issue. I guess since I already wrote this one I don't want to throw away the testing of  `merge-requests`.

Agree it isn't the simplest test for sure! Took me awhile to get it all going.

WDYT?

Reply:
True, makes sense too. was just thinking as this had to re-create some of the MR creation logic. Let's leave it as is then :)